pkgs/top-level/all-packages.nix

```diff
@@ -7086,6 +7086,8 @@ with pkgs;
 
   zoxide = callPackage ../tools/misc/zoxide { };
 
+  zeekscript = callPackage ../tools/security/zeekscript { };
```

Comment:
Can you move this to the alphabetically correct location please? (should come right after `zeek`)